### PR TITLE
(modify): add secrets in disk-loss chart

### PR DESCRIPTION
**Pull request by ispeakc0de**
Signed-off-by: shubhamchaudhary <shubham.chaudhary@mayadata.io>

-  Add secrets in disk loss experiment

-  Add some default ENV variables

### CODE REVIEW

File: charts/generic/disk-loss/experiment.yaml [hunk 1]
```diff
@@ -32,8 +32,40 @@ spec:
...
-      value: 15
+      value: '15'
+
+    - name: APP_CHECK
+      value: 'true' 
+
+    - name: CHAOS_NAMESPACE
+      valueFrom:
+        fieldRef:
+          fieldPath: metadata.namespace
```
Comment (ksatchit) [inline charts/generic/disk-loss/experiment.yaml]: make this also key-value

**Review by chandankumar4 — DISMISSED** [review]
lgtm

File: charts/generic/disk-loss/experiment.yaml [hunk 2]
```diff
@@ -32,8 +32,35 @@ spec:
...
+      value: ''  
+          
+    - name: ZONE_NAME
+      value: '' 
+          
     labels:
       name: disk-loss
+    secrets:
+    - name: disk-loss
+      mountPath: /mnt/cloud_config
```
Comment (ksatchit) [inline charts/generic/disk-loss/experiment.yaml]: Directly /mnt/cloud_config.yml?

File: charts/generic/disk-loss/experiment.yaml [hunk 3]
```diff
@@ -24,16 +24,43 @@ spec:
     verbs:
       - "*"
   definition:
-    image: "litmuschaos/ansible-runner:ci"
+          image: "litmuschaos/ansible-runner:ci"
```
Comment (ksatchit) [inline charts/generic/disk-loss/experiment.yaml]: indent?

**Pull request merged**